File: Programming Basics with Python/T03 - Conditional Statements/exercise/06. World Swimming Record.py
record_sec = float(input())
distance_meters = float(input())
seconds_for_meter = float(input())

water_density = 15
water_slow = 12.5

diff = 0
total_time = distance_meters * seconds_for_meter

delay = distance_meters // water_density * water_slow
total_time = total_time + delay

if total_time < record_sec:
    print(f'Yes, he succeeded! The new world record is {total_time:.2f} seconds.')
else:
    diff = total_time - record_sec
    print(f'No, he failed! He was {diff:.2f} seconds slower.')

File: Programming Basics with Go/T03 - Conditional Statements/exercise/06. World Swimming Record.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var recordTime float32
	fmt.Scanln(&recordTime)

	var distance float32
	fmt.Scanln(&distance)

	var timePerMeter float32
	fmt.Scanln(&timePerMeter)

	var swimTime = distance * timePerMeter
	var additionalTime = math.Floor(float64(distance) / 15.0) * 12.5
	var totalTime = swimTime + float32(additionalTime)

	if (totalTime < recordTime) {
		fmt.Printf("Yes, he succeeded! The new world record is %.2f seconds.", totalTime)
	} else {
		fmt.Printf("No, he failed! He was %.2f seconds slower.", totalTime - recordTime)
	}
}